tests/make/Makefile

```diff
@@ -1 +1,3 @@
+export YAMLLINT_CONFIG := ../../.yamllint.yml
```

Comment:
Should probably change this to a `?=` so that someone could pass in their own yaml lint config if they wanted to.

Reply:
technically [it is](https://github.com/plus3it/tardigrade-ci/pull/21/files#diff-b67911656ef5d18c4ae36cb6741b7965R157) I'm enforcing this var value here b/c the expectation is that this makefile will always include the 'upstream' makefile. This Makefile is here to make implementing the bats tests easier. More specifically its so I don't have to manage the bats `make` commands on a test by test basis.